gluu-server/opt/gluu/jetty/oxauth/custom/pages/WEB-INF/incl/content/nrcan_geo/help-en.xhtml

```diff
@@ -0,0 +1,33 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<!DOCTYPE html>
+<ui:composition xmlns="http://www.w3.org/1999/xhtml" xmlns:ui="http://xmlns.jcp.org/jsf/facelets">
```

Comment:
We haven't received the content from NRCan yet. They have now missed the deadline for providing it for this release, and we don't plan another until September.

With these placeholder files it will be slightly easier to add their content manually when it eventually arrives.